libraries/AC_Avoidance/AC_Avoid.h

```diff
@@ -200,8 +209,11 @@ class AC_Avoid {
     AP_Float _margin;           // vehicle will attempt to stay this distance (in meters) from objects while in GPS modes
     AP_Int8 _behavior;          // avoidance behaviour (slide or stop)
     AP_Float _backup_speed_max; // Maximum speed that will be used to back away (in m/s)
+    AP_Float _min_alt;          // alt below which Proximity based avoidance is turned off
+    AP_Float _accel_max;        // maximum accelration while simple avoidance is active
 
     bool _proximity_enabled = true; // true if proximity sensor based avoidance is enabled (used to allow pilot to enable/disable)
+    bool _alt_proximity_enabled = true; // true if proximity sensor based avoidance is enabled based on altitude
```

Comment:
maybe let's rename "_alt_proximity_enabled" to "_proximity_alt_enabled".  it's a bit of an odd name but I'd like to keep proximity at the front.